operations/k6/load-testing-with-k6.js

```diff
@@ -89,19 +89,30 @@ const HA_REPLICAS = parseInt(__ENV.K6_HA_REPLICAS || 1);
  * @constant {number}
  */
 const HA_CLUSTERS = parseInt(__ENV.K6_HA_CLUSTERS || 1);
+/**
+ * Allow for this script to write to a specific tenant in the Mimir cluster.
+ * By default, no tenant is specified requiring the cluster to have multi-tenancy disabled.
+ * @constant {string}
+*/
+const TENANT_NAME = __ENV.K6_TENANT_NAME || '';
```

Comment:
Do you mind calling it `TENANT_ID` to keep it consistent with Mimir, please?

Reply:
Thanks for taking a look. No, I don't mind. Will push something in a moment